web-app/src/test/java/com/google/sps/servlets/ReportServletTest.java

```diff
@@ -19,26 +24,39 @@
 @RunWith(JUnit4.class)
 public class ReportServletTest extends Mockito {
 
+  private final LocalServiceTestHelper helper =
+  new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig(), new LocalBlobstoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
   @Test
   public void testReportServlet() throws IOException {
-    HttpServletRequest request = mock(HttpServletRequest.class);
-    HttpServletResponse response = mock(HttpServletResponse.class);
-
-    when(request.getParameter("title")).thenReturn("Test");
-    when(request.getParameter("latitude")).thenReturn("1.11");
-    when(request.getParameter("longitude")).thenReturn("10.26");
-    when(request.getParameter("timestamp")).thenReturn("2017-06-01T08:30");
-    when(request.getParameter("incidentType")).thenReturn("Theft");
-    when(request.getParameter("description")).thenReturn("Sample request for testing");
-
-    Entity testReport = createReportEntity(request, response);
-
-    Assert.assertEquals(testReport.getProperty("title"), "Test");
-    Assert.assertEquals(testReport.getProperty("latitude"), 1.11);
-    Assert.assertEquals(testReport.getProperty("longitude"), 10.26);
-    Assert.assertEquals(testReport.getProperty("incidentType"), "Theft");
-    Assert.assertEquals(testReport.getProperty("description"), "Sample request for testing");
-    Assert.assertEquals(testReport.getProperty("timestamp"), 1496273400);
+    // HttpServletRequest request = mock(HttpServletRequest.class);
```

Comment:
nit: either uncomment this, or delete this test completely. Since this PR just fixes the merge, I would just uncomment this and make the test pass in another PR.